compiler-rt/lib/ctx_profile/CtxInstrProfiling.cpp

```diff
@@ -27,6 +29,20 @@ __sanitizer::SpinMutex AllContextsMutex;
 SANITIZER_GUARDED_BY(AllContextsMutex)
 __sanitizer::Vector<ContextRoot *> AllContextRoots;
 
+__sanitizer::atomic_uintptr_t AllFunctionsData = {};
```

Comment:
nit: Prefer AllFunctionsData{0}

I think semantically these mean the same thing, just a matter of preference.

Reply:
I did `= something` elsewhere in this file

Reply:
Not sure I understand what you mean?

Reply:
I use the `= {}` notation elsewhere in this file and the .h.